aptos-move/framework/aptos-stdlib/sources/simple_map.move

```diff
@@ -183,6 +199,26 @@ module aptos_std::simple_map {
         destroy_empty(map);
     }
 
+    #[test]
+    public fun test_keys() {
+        let map = create<u64, u64>();
+        assert!(keys(&map) == vector[], 0);
+        add(&mut map, 2, 1);
+        add(&mut map, 3, 1);
+
+        assert!(keys(&map) == vector[2, 3], 0);
+    }
+
+    #[test]
+    public fun test_values() {
+        let map = create<u64, u64>();
+        assert!(values(&map) == vector[], 0);
+        add(&mut map, 2, 1);
+        add(&mut map, 3, 1);
+
+        assert!(values(&map) == vector[1, 1], 0);
```

Comment:
Can you test with difference values? In this case we cannot tell if all 1 come from the same entry.

Reply:
Done